recipes/msaprobs/build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+mkdir -p $PREFIX/bin
+(cd $SRC_DIR/MSAProbs && make all)
```

Comment:
Since the `build.sh` script starts in the unpacke source dir, you can simplify this and the next line slightly to
```
cd MSAProbs
make all
cp msaprobs $PREFIX/bin/
```
Note also that the build script is called with the Bash option `-o errexit`, which means that you don’t need to connect commands with `&&`.